packages/plugins/FileService/package.json

```diff
@@ -22,12 +22,17 @@
     "@masknet/typed-message-react": "workspace:^",
     "@masknet/web3-shared-base": "workspace:^",
     "@msgpack/msgpack": "^2.8.0",
+    "@remix-run/router": "^1.0.4",
```

Comment:
Why not use `react-router-dom`?

Reply:
Only `react-router-dom` provides `InitialEntry` for `initialEntries` of `MemoryRouter`